src/main/java/jacchm/footballapp/exception/ControllerAdvisor.java

```diff
@@ -0,0 +1,26 @@
+package jacchm.footballapp.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.time.LocalDateTime;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@ControllerAdvice
+public class ControllerAdvisor extends ResponseEntityExceptionHandler {

```

Comment:
Wiem, że to kłócenie się o nazwy, ale lepiej tego advice nazwać GlobalExceptionHandler - bo to w sumie jego rola